app/uk/gov/hmrc/soletraderidentificationfrontend/views/templates/error_template.scala.html

```diff
@@ -1,22 +1,22 @@
 @*
- * Copyright 2021 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
-
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
 @import uk.gov.hmrc.soletraderidentificationfrontend.config.AppConfig
 @import uk.gov.hmrc.soletraderidentificationfrontend.models.PageConfig
 @import uk.gov.hmrc.soletraderidentificationfrontend.views.html.helpers._
+@import uk.gov.hmrc.soletraderidentificationfrontend.views.html.templates.govuk_template
```

Comment:
Do we need this? I get an error saying it's being ignored when compiling?

Reply:
We dont because it is in the same package but my intellij ultimate edition shows an error if I dont put it. It suggested to put ....  I think the Play Plugin has some problem.
I will remove it.
<img width="1580" alt="Screenshot 2022-01-05 at 09 35 32" src="https://user-images.githubusercontent.com/4958044/148195287-03762c1a-6e1b-40bd-9380-046cf84fd4b0.png">
<img width="1628" alt="Screenshot 2022-01-05 at 09 33 39" src="https://user-images.githubusercontent.com/4958044/148195307-45fc7d0b-8b14-41ac-bd57-8c5da085313e.png">
.